index.html

```diff
@@ -5,13 +5,143 @@
+                <p class="banner-text">WORLD HEALTH SUMMIT 2022</p>
+                <p class="moto">WHS2022 - where science, politics, the private sector and civil society meet for
+                    inspiring talks, enhanced cooperations and new solutions.</p>
+                <p class="date">2022.02.16(WED) ~ 18(FRI)</p>
+                <p class="venue">@ Hotel Berlin Central District, Berlin, Germany</p>
+            </div>
+        </div>
+    </section>
+    <section id="main-program">
+        <div class="main-program-container">
+            <div class="section-title-style">
+                <h1>Main Program</h1>
+                <div class="underline-section"></div>
+            </div>
+            <div class="main-program">
+                <div class="main-program-card">
+                    <img src="img/ic-mp1.png" alt="M8 Aliance image">
+                    <h4>M8 Aliance</h4>
+                    <p>Migrant and Refugee Health has been a M8 Alliance priority topic since 2015. It featured high on
+                        the World Health Summit.</p>
+                </div>
+                <div class="main-program-card">
+                    <img src="img/ic-mp2.png" alt="Pandemic Preparedness image">
+                    <h4>Pandemic Preparedness</h4>
+                    <p>Migrant and Refugee Health has been a M8 Alliance priority topic since 2015. It featured high on
+                        the World Health Summit.</p>
+                </div>
+                <div class="main-program-card">
+                    <img src="img/ic-mp3.png" alt="Investment in Health image">
+                    <h4>Investment in Health</h4>
+                    <p>Migrant and Refugee Health has been a M8 Alliance priority topic since 2015. It featured high on
+                        the World Health Summit.</p>
+                </div>
+                <div class="main-program-card">
+                    <img src="img/ic-mp4.png" alt="Digital Transformation">
+                    <h4>Digital Transformation</h4>
+                    <p>Migrant and Refugee Health has been a M8 Alliance priority topic since 2015. It featured high on
+                        the World Health Summit.</p>
+                </div>
+                <div class="main-program-card">
+                    <img src="img/ic-mp5.png" alt="Polio-free Future">
+                    <h4>Polio-free Future</h4>
+                    <p>Migrant and Refugee Health has been a M8 Alliance priority topic since 2015. It featured high on
+                        the World Health Summit.</p>
+                </div>
+            </div>
+            <div>
+                <a href="#">SEE THE WHOLE PROGRAM</a>
+            </div>
+        </div>
+    </section>
+
+    <!-- Dynamic html loads here -->
+    <section id="featured-speakers">
+        <div class="section-title-style">
+            <h1>Featured Speakers</h1>
+            <div class="underline-section"></div>
+        </div>
```

Comment:
- [ ] The featured speaker's section images do not match the original, kindly fix it. Take a look at the photos below.
- [ ] The font size and font weight do not match the original, kindly work on that as well. 

### Yours

![yours](https://user-images.githubusercontent.com/53124548/176977993-a2e0767f-83ea-47c0-84b5-97df22cb3131.png)

### Original

![original](https://user-images.githubusercontent.com/53124548/176978017-d5702592-88e2-470a-8e9b-f6dfd9dbefc4.png)

Reply:
@elisha2kyakpo1
- I have updated the live demo. It was pointing to the previous branch.
- Fix font size difference